plugins/orchestrator-backend/src/types/SonataWorkFlow.ts

```diff
@@ -0,0 +1,9 @@
+export interface SonataWorkFlow {
```

Comment:
instead of sonataflow we can call this as WorkflowDefinition but we already have an interface for it in the file Guilherme pointed out before, so maybe we can reuse adding the missing attributes.
https://github.com/caponetto/backstage-plugins/blob/cb4bc4f3f8ba0fff906ec0635f6441c99deb1f83/plugins/orchestrator-common/src/types.ts#L23